dojo/tasks.py

```diff
@@ -86,6 +88,29 @@ def cleanup_alerts(*args, **kwargs):
         logger.info('total number of alerts deleted: %s', total_deleted_count)
 
 
+@app.task(bind=True)
+def flush_auditlog(*args, **kwargs):
+    try:
+        retention_period = settings.AUDITLOG_FLUSH_RETENTION_PERIOD
+    except System_Settings.DoesNotExist:
+        retention_period = -1
```

Comment:
```suggestion
    retention_period = settings.AUDITLOG_FLUSH_RETENTION_PERIOD
```
This will never happen. `System_Settings != django.conf.settings`. If you want to set default `AUDITLOG_FLUSH_RETENTION_PERIOD` to `-1`, please to it in `dojo.settings.settings`